terraform/pagerduty/modules/service/technical/main.tf

```diff
@@ -275,3 +275,34 @@ resource "aws_secretsmanager_secret_version" "pagerduty_airflow_integration_key"
   secret_id     = aws_secretsmanager_secret.pagerduty_airflow_integration_key[0].id
   secret_string = pagerduty_service_integration.airflow[0].integration_key
 }
+
+##################################################
+# Alert Manager Integration
+##################################################
+
+data "pagerduty_vendor" "alert_manager" {
+  name = "Prometheus"
+}
+
+resource "pagerduty_service_integration" "alert_manager" {
+  count = var.enable_alert_manager_integration ? 1 : 0
+
+  name    = data.pagerduty_vendor.alert_manager.name
+  service = pagerduty_service.this.id
+  #vendor = data.pagerduty_vendor.alert_manager.id
```

Comment:
Could you please annotate why this is currently commented out? We will forget. :)